library/decls.mli

```diff
@@ -34,12 +34,3 @@ val variable_exists : variable -> bool
 
 val add_constant_kind : Constant.t -> logical_kind -> unit
 val constant_kind : Constant.t -> logical_kind
```

Comment:
This `constant_kind` seems to be used only for `Typeclasses.is_instance` which has no caller and really weird semantics (look at if if you're curious). Maybe we should just remove them (constant_kind and is_instance).

Reply:
Oh constant_kind is also used in dumpglob sorry. is_instance should definitely be removed though.